Android/quickstart-kotlin/app/src/main/assets/immersiveReader.html

```diff
@@ -0,0 +1,34 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. -->
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <script type="text/javascript" src="https://contentstorage.onenote.office.net/onenoteltir/immersivereadersdk/immersive-reader-sdk.0.0.3.js"></script>
+</head>
+<body>
+    <script type="text/javascript">
+        function handleLaunchImmersiveReader(message) {
+            if (!message) {
+                Android.showToast('Message is null or undefined!');
+            } else {
+
+                // A simple declarative function used to close the Immersive Reader WebView via @JavaScriptInterface
+                const exitCallback = function() {
+                    Android.immersiveReaderExit();
+                }
+
+                // A method cannot be passed into the message JSON, so we assign it here as there's
+                // no need to post a message to the parent of the Immersive Reader iframe because
+                // the host application uses a WebView.
+                message.options.onExit = exitCallback;
+
+                // Use the JavaScript SDK to launch the Immersive Reader.
+                ImmersiveReader.launchAsync(message.cogSvcsAccessToken, message.cogSvcsSubdomain, message.request, message.options);

```

Comment:
can we change message.request to message.content in order to be consistent with the parameter names? https://github.com/microsoft/immersive-reader-sdk/blob/master/js/src/launchAsync.ts